PowerPointLabs/Test/Util/PathUtil.cs

```diff
@@ -21,7 +21,12 @@ public static String GetParentFolder(String path, int loopCount)
 
         public static String GetTempPath(String fileName)
         {
-            return Path.GetTempPath() + fileName;
+            return GetTempTestFolder() + fileName;
+        }
+
+        public static String GetTempTestFolder()
```

Comment:
You can make class public for PowerPointLabs.Utils.PathUtil, and then re-use it here